### PR TITLE
Tau Ceti Legion Uniform Improvement

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by Ben10083 — CHANGES_REQUESTED** [review]
With you deleting the older uniforms, you need to modify the TCFL ghost roles so they spawn with the right clothing, and replace some of the legion clothing on their ship.

**Comment by alsoandanswer** [comment]
> With you deleting the older uniforms, you need to modify the TCFL ghost roles so they spawn with the right clothing, and replace some of the legion clothing on their ship.

I didn't delete anything. I just changed the icon_state.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -505,29 +505,31 @@
 /obj/item/clothing/under/legion
 	name = "Tau Ceti Foreign Legion uniform"
 	desc = "A blue field uniform worn by Tau Ceti Foreign Legion forces."
-	icon_state = "taucetilegion"
+	icon = 'icons/clothing/under/uniforms/tcfl_uniform.dmi'
+	icon_state = "tauceti_volunteer"
 	item_state = "bl_suit"
-	worn_state = "taucetilegion"
+	worn_state = "tauceti_volunteer"
```
Comment (alsoandanswer) [inline code/modules/clothing/under/miscellaneous.dm:511]:
```suggestion
	worn_state = "tauceti_volunteer"
	contained_sprite = TRUE
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -505,29 +505,32 @@
 /obj/item/clothing/under/legion
 	name = "Tau Ceti Foreign Legion uniform"
 	desc = "A blue field uniform worn by Tau Ceti Foreign Legion forces."
-	icon_state = "taucetilegion"
+	icon = 'icons/clothing/under/uniforms/tcfl_uniform.dmi'
+	icon_state = "tauceti_volunteer"
 	item_state = "bl_suit"
```
Comment (alsoandanswer) [inline code/modules/clothing/under/miscellaneous.dm:510]:
```suggestion
```

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**